python/__init__.py

```diff
@@ -19,32 +19,30 @@
 # The presence of this file turns this directory into a Python package
 
 '''
-This is the GNU Radio DAB module. Place your Python package
-description here (python/__init__.py).
+This is the GNU Radio DAB module.
 '''
 
+# The presence of this file turns this directory into a Python package
+import os
+
 # import swig generated symbols into the dab namespace
 try:
-	# this might fail if the module is python-only
-	from dab_swig import *
+    # this might fail if the module is python-only
+    from dab_swig import *
```

Comment:
Well, if you touch these lines, you might as well remove the `try/except ImportError: pass`, as this module is definitily **not** python-only.